lib/instrumentation/https.test.js

```diff
@@ -157,7 +157,7 @@ test("node 10.9.0+ url + options", (done) => {
   tracker.setTracked(newMockContext());
 
   https.get(
-    "https://google.com:80",
+    "https://google.com:443",
```

Comment:
Same hunch as the `http` tests; I think they're testing priority on what sets the port.